src/cosalib/qemuvariants.py

```diff
@@ -304,24 +304,30 @@ def mutate_image(self):
             log.info(f"Moving {work_img} to {final_img}")
             shutil.move(work_img, final_img)
 
-        if self.gzip:
+        if self.compression == "skip":
+            meta_patch.update({
+                'skip-compression': True
+            })
+        elif self.compression is not None:
             sha256 = sha256sum_file(final_img)
             size = os.stat(final_img).st_size
             temp_path = f"{final_img}.tmp"
+
+            match self.compression:
+                case "gzip":
+                    rc = ['gzip', '-9c', final_img]
+                case "zip":
+                    rc = ['zip', '-9', "-", final_img]
+                case default:
```

Comment:
Apparently CI doesn't like this now.  Looking at Python docs, I guess it should be:
```suggestion
                case _:
```

Reply:
fixed up